config/application.rb

```diff
@@ -11,7 +11,7 @@
 APPLICATION_ENVS = YAML.load_file('config/application.yml').keys - ['development', 'test'] rescue []
 APPLICATION_ENVIRONMENT_VARIABLES = YAML.load(ERB.new(File.read('config/application.yml')).result)
 
-MFA_ENABLED = ActiveModel::Type::Boolean.new.cast(APPLICATION_ENVIRONMENT_VARIABLES[Rails.env]['MFA_ENABLED']) rescue false
+MFA_ENABLED = ActiveModel::Type::Boolean.new.cast(APPLICATION_ENVIRONMENT_VARIABLES[Rails.env]['MFA_ENABLED']) || ENV['MFA_ENABLED'] rescue false
```

Comment:
damn that's a complicated line ^^

`ENV['MFA_ENABLED'] == 'true'` wasn't doing the job?

Reply:
We could drop the first line. It's looking for production: mfa_enabled in application.yml or using the ENV['MFA_ENABLED']. It's because the environments are strange in the manager :/.

Reply:
I guess you don't enable MFA only for a staging or production in the manager. It's not worker related.